lib/ocpp/v16/charge_point_configuration.cpp

```diff
@@ -262,6 +262,18 @@ void ChargePointConfiguration::init_supported_measurands() {
     }
 }
 
+void ChargePointConfiguration::setChargepointInformation(const std::string& chargePointVendor,
```

Comment:
Please add other properties of the BootNotification as well:
* iccid
* imsi
* meterSerialNumber
* meterType
* chargeBoxSerialNumber

Reply:
ok, will add as std::optional parameter